Validation/RecoB/plugins/BDHadronTrackMonitoringAnalyzer.cc

```diff
@@ -38,6 +38,7 @@ BDHadronTrackMonitoringAnalyzer::BDHadronTrackMonitoringAnalyzer(const edm::Para
   TrackCollectionTag_ = consumes<reco::TrackCollection>(TrackSrc_);
   PrimaryVertexColl_ = consumes<reco::VertexCollection>(PVSrc_);
   clusterTPMapToken_ = consumes<ClusterTPAssociation>(ClusterTPMapSrc_);
+  ttrackToken_ = esConsumes();
```

Comment:
```suggestion
  ttrackToken_ = esConsumes(edm::ESInputTag("","TransientTrackBuilder"));
```

a label used to be specified at line 217, no?

Reply:
Yes. Thanks! I have fixed it.